File: ios/Classes/Bridge/GetControllerIdBridge.swift
import Usercentrics

struct GetControllerIdBridge : MethodBridge {

    let name: String = "getControllerId"
    let usercentrics: UsercentricsProxyProtocol

    func invoke(_ call: FlutterMethodCall, _ result: @escaping FlutterResult) {
        assert(call.method == name)
        result(usercentrics.shared.getControllerId())
    }
}

File: android/src/main/kotlin/com/usercentrics/sdk/flutter/bridge/GetControllerIdBridge.kt
package com.usercentrics.sdk.flutter.bridge

import android.app.Activity
import com.usercentrics.sdk.Usercentrics
import com.usercentrics.sdk.flutter.extension.FlutterResult
import io.flutter.plugin.common.MethodCall

internal class GetControllerIdBridge : MethodBridge {

    override val name: String
        get() = "getControllerId"

    override fun invoke(call: MethodCall, result: FlutterResult, activity: Activity?) {
        result.success(Usercentrics.instance.getControllerId())
    }

}